libs/application/ui-fields/src/lib/SliderFormField/SliderFormField.tsx

```diff
@@ -20,6 +20,26 @@ type SliderFormFieldProps = {
 export const SliderFormField: FC<
   React.PropsWithChildren<SliderFormFieldProps>
 > = ({ application, field }) => {
+  const {
+    id,
+    min,
+    max,
+    title,
+    trackStyle,
+    calculateCellStyle,
+    showLabel,
+    showMinMaxLabels,
+    showRemainderOverlay,
+    showProgressOverlay,
+    showToolTip,
+    label,
+    rangeDates,
+    onChangeEnd,
+    labelMultiplier,
+    snap,
+    step,
+    saveAsString,
+  } = field
```

Comment:
You are spreading the values here from the field still referencing the field for everything other than "max"; won't this break?

Reply:
No it won't break, the values were just unused. Fixed now :)